src/utils/prices.ts

```diff
@@ -23,9 +23,14 @@ export function computeRealizedPriceImpact(trade: InterfaceTrade): Percent {
   return trade.priceImpact.subtract(realizedLpFeePercent)
 }
 
-export function getPriceImpactWarning(priceImpact: Percent): 'warning' | 'error' | undefined {
+/**
+ * Returns the warning level for a given % change. The change can be either positive or negative.
```

Comment:
lets figure out the +- price impact issue first before committing this comment to code because it could be confusing

and also we should probably decouple using this helper function for both price impact and for fiat value change

Reply:
sounds good, removed from this PR